engine/classes/Elgg/Database/Select.php

```diff
@@ -11,6 +11,10 @@ class Select extends QueryBuilder {
 	 * {@inheritdoc}
 	 */
 	public static function fromTable($table, $alias = null) {
```

Comment:
why not default in the function params? (eg. `$alias = 'master'`)

Reply:
Someone could pass `false` or `''`

Reply:
then they 'know' what they're doing and maybe don't expect 'master'?